src/main/java/net/minestom/server/instance/Instance.java

```diff
@@ -279,6 +280,14 @@ public void unloadChunk(int chunkX, int chunkZ) {
         unloadChunk(chunk);
     }
 
+    public void invalidateSection(int sectionX, int sectionY, int sectionZ) {
+        final Chunk chunk = getChunk(sectionX, sectionZ);
+        if (chunk != null) {
```

Comment:
we need to reevaluate the heightmap (needsCompleteHeightmapRefresh in DynamicChunk) in DynamicChunk#invalidate also (probably a mistake that its not there already, but exposed by this pr)